analysis/wwz/make_datacards.py

```diff
@@ -259,13 +259,20 @@ def handle_negatives(in_dict):
             val = in_dict[cat]["nominal"][proc][0]
             var = in_dict[cat]["nominal"][proc][1]
             if val <= 0:
-                print(f"WARNING: Process \"{proc}\" in cat \"{cat}\" is negative ({val}), replacing with {SMALL} and shifting up/down systematic variations accordingly.")
+                #!!!OLD METHOD FOR HANDLING LOW YIELD BINS!!!
```

Comment:
What does this refer to? "OLD" meaning what? It all seems to be added code in this PR. Is the commented part needed?

Reply:
this is our previous method for handling small (negative) yields. We would set to SMALL and then center the systematics around SMALL. It was in the code previously

Reply:
But this code is new. The green indicates new compared to the master

Reply:
The removed lines above and below are the original code. This is registered as new code even though it is just commented out

Reply:
Could you remind me exactly what the old procedure was and what the new procedure is?

I think that either we want to remove the old entirely, or make a switch between them. 

But whichever of those two options we go with, I think we do not want to leave the old dead commented code hanging around, it will just confuse future us.

Reply:
Our old method took negative yields and set them to zero. Then recentered the systematics around zero. 

However, since we want the zero_low_mc to be an option when we run the datacard maker. I have made it so the old option will be used when not zeroing low mc bins and the new one will be used when we are